django_project/.env.template

```diff
@@ -0,0 +1,13 @@
+DEBUG=True
+SECRET_KEY='my_sycret_key'
+ALLOWED_HOSTS='127.0.0.1, localhost'
+PORT=8000
+
+SQLITE3_DB_NAME='db'
```

Comment:
👍

Reply:
But, as I remember, better to use DATABASE_URL, so it will be isolated of implementation